docs/source/support/release-notes.rst

```diff
@@ -6,9 +6,158 @@ Release notes
+-  ``daml damlc inspect-dar`` now has a ``--json`` flag to produce
+   machine-readable output. See
+   the `documentation <https://docs.daml.com/1.2.0/daml/reference/packages.html#inspecting-dars>`__
+   for more information.
+-  The Scala bindings have gained a method, ``LedgerClient#close``, which
+   will shut down the channel and await termination. This is
+   optional; the channel will still be shut down on JVM exit if this
+   method is not called.
+-  Record dot syntax like ``rec.field1.field2`` is now handled in
+   expressions entered into the REPL.
+-  ``daml trigger``, ``daml script`` and ``daml repl`` now all support
+   the ``--max-inbound-message-size`` command line flag, which configures
+   the maximum size of transactions that can be handled.
+-  The ``createAndExerciseCmd`` command has been added to DAML Triggers.
+
+Security and Bugfixes
+~~~~~~~~~~~~~~~~~~~~~
+
+-  Dependencies have been upgraded to newer versions to avoid
+   exposure to reported security vulnerabilities.
+
+   -  Upgrade ``jackson`` version to ``2.11.0`` from ``2.9.9.3``
+   -  Upgrade ``io.grpc:grpc-xxxxx`` and ``io.netty:netty-xxx`` version
+      to latest
+   -  Upgrade ``protobuf`` and ``protobuf-java`` to 3.11.0
+
+-  A Sandbox Classic migration issue when used with postgres has been
+   fixed.
+   See `#6017 <https://github.com/digital-asset/daml/issues/6017>`__
+-  A bug where large multi-command transactions would cause a stack
+   overflow in DAML Script was fixed.
+-  The Standard Library’s ``DA.Text.splitOn`` function will now correctly
+   handle the case where the separator appears at the end but should
+   not be matched, as in ``splitOn "aa" "aaa" == ["", "a"]``.
+   See `#5786 <https://github.com/digital-asset/daml/issues/6017>`__ for
+   more details.
+-  The DAML linter, dlint, has been improved by removing some
+   Haskell-based rules not currently applicable to DAML and by
+   changing some function references.
+
+Ledger Integration Kit
+~~~~~~~~~~~~~~~~~~~~~~
+
+-  The Ledger API Server emits new metrics for the LF Value Cache. If
+   the ``--max-state-value-cache-size`` is greater than zero, the
+   following additional metrics will be recorded under the
+   ``daml.kvutils.submission.validator.state_value_cache`` namespace: 
+
+   -  ``hits``
+   -  ``misses``
+   -  ``load_successes``
+   -  ``load_failures``
+   -  ``load_total_time``
+   -  ``evictions``
+   -  ``evicted_weight``
+
+-  Added new Ledger API Server metrics
+   for ``daml.index.db.*.translation`` to measure the time spent
+   translating to and from the serialized DAML-LF values when fetched
+   from the participant index.
+-  Added new Ledger API Server metrics
+   for ``daml.index.db.*.deserialization`` to measure the duration of the
+   translation of the serialized DAML-LF values when fetched from the
+   participant index.
+-  The Ledger API Test Tool has gained
+   the ``TransactionSize`` performance benchmark test.
+
+What’s Coming
```

Comment:
I don't think this should go into our docs, but just the blog.

Reply:
We always included it in the past. Happy to stop doing it from now on if you want.

Reply:
removed